fdbrpc/RangeMap.h

```diff
@@ -82,25 +92,24 @@ class RangeMap {
 			//ASSERT( it->key != allKeys.end );
 			return it->value;
```

Comment:
This `value()` method should return `const Val&` if `isConst` is true.

Reply:
Done.